### DISCUSSION TITLE
vpn

### DISCUSSION THREAD

**Discussion by ttigero** [linuxdeepin/developer-center#4790] [Troubleshooting | 故障排查]
There is a problem with vpn connection , although the settings have been set correctly, but the connection is rejected in the vpn connection

**Comment by BLumia** [comment]
Cannot reproduce this issue. Mind to provide more details?

**Comment by ttigero** [comment]
I have two computers one of them installed deepin 20.9 , and another deepin 23 , i have vpn service from openVpn and it configured on deepin 20.9 it is working without problem , i have export config file from deepin 20.9 to import file on deepin 23 but it reject to connect!!!

**Reply by BLumia** [reply]
> but it reject to connect!!!

Any error logs (e.g. from `journalctl -u NetworkManager.service`) ? What if you try to connect them using network-manager directly (e.g. from `nmtui`)? What openvpn configuration are you using? I am also using openvpn under v23 and it works without issue so detail information is needed.

**Reply by ttigero** [reply]
yes , i am using nmtui

**Reply by BLumia** [reply]
So any error logs?
